### ISSUE TITLE
[rancher-logging] Support enhanceK8s filter plugin

### ISSUE THREAD

**Issue by dbason** [rancher/rancher#35223]
With Opni we are making use of the enhanceK8s filter plugin for fluentd to add additional k8s metadata to the logs, and would like for this to be supported in rancher-logging.

To enabled the required RBAC to be set up automatically by the operator enhanceK8s needs to be added as a global filter in the LoggingSpec.

The fluentd image will need to be updated to 1.13.9 version, however there is a problem with the detect exceptions plugin in that version - https://github.com/banzaicloud/logging-operator/issues/861.  I've currently worked around this by building a custom fluentd image that uses the 0.0.13 version of the gem.

**Comment by dbason** [comment]
@jiaqiluo is there a label we can add to stop the issue going stale?

**Comment by jiaqiluo** [comment]
yep, there is a list of them: https://github.com/rancher/rancher/blob/725a0b4702670616bc5dff507bd9fec0720676c0/.github/stale.yml#L13-L23
